source/common/buffer/buffer_impl.h

```diff
@@ -409,7 +407,7 @@ class Slice {
    * accessed directly; access base_ instead. */
   StoragePtr storage_;
 
-  /** Start of the slice. Points to storage_ iff the slice owns its own storage. */
+  /** Start of the slice. Points to storage_ if the slice owns its own storage. */
```

Comment:
revert: https://www.merriam-webster.com/dictionary/iff

Reply:
get it.